PocketKit/Sources/PocketKit/MyList/ItemsList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -4,14 +4,26 @@ import Analytics
 import Combine
 import UIKit
 
+public enum SavesViewType {
+    case saves
+    case archive
```

Comment:
just an FYI - we do have other existing `enums` not sure if we should repurpose them or create a new one.